Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
```

Comment:
Thanks for the explanation. Haven't tried it yet, but seems like it makes sense to bring this into the codebase. I am assuming there's no security issues with bringing this into the codebase?